mantle/kola/harness.go

```diff
@@ -60,6 +60,12 @@ import (
 	"github.com/coreos/mantle/util"
 )
 
+// InstalledTestsDir is a directory where "installed" external
+// can be placed; for example, a project like ostree can install
+// tests at /usr/lib/coreos-assembler/tests/kola/ostree/...
+// and this will be automatically picked up.
+const InstalledTestsDir = "/usr/lib/coreos-assembler/tests/kola"
```

Comment:
nit: could we make this a var instead, e.g.

```
const defaultInstalledTestDirs = "/usr/lib/coreos-assembler/tests/kola"
var InstalledTestsDir = defaultInstalledTestDir
```
Such that ` go run -ldflags '-X kola.harness.InstalledTestsDir="foo" ...'` works?

Reply:
The idea is that this is a standard directory for other components to install to; we already have a way to do "find tests from elsewhere" via `-E` right?